sys-apps/hd-idle/files/hd-idle-init

```diff
@@ -0,0 +1,19 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
```

Comment:
There's a number of missing variables in this file. Please take a look at https://gitweb.gentoo.org/repo/gentoo.git/tree/sys-cluster/zetcd/files/zetcd.initd if you need some inspiration and make sure to read `openrc-run`'s manpage :)

Reply:
Since hd-idle does not daemonize (and not create PID files), I have now added a minimal set of variables and removed the custom start and stop functions. Let me know if there are variables which are required. 
Looks much nicer and shorter now 👍  Thanks!